### ISSUE TITLE
Background apps placement

### ISSUE THREAD

**Issue by rickdkk** [eonpatapon/gnome-shell-extension-caffeine#253]
The toggle for caffeine sometimes ends up below the background apps toggle. I personally think this is not aesthetically pleasing. In GSConnect they solved this by adding:

```javascript
       // Ensure the tile(s) are above the background apps menu
        for (const item of this.quickSettingsItems) {
            QuickSettingsMenu.menu._grid.set_child_below_sibling(item,
                QuickSettingsMenu._backgroundApps.quickSettingsItems[0]);
        }
```

in the init of the systemindicator. 

How do y'all feel about this?

**Comment by stuarthayhurst** [comment]
Already fixed in #250, thanks for the report :)

**Comment by rickdkk** [comment]
Oops, forgot to check the closed issues. Thanks for the swift reply and the great extension!
